src/index.js

```diff
@@ -0,0 +1,23 @@
+import './style.css';
+import fetchMeals from './module/display.js';
+
+fetchMeals();
+
+// // Render page by search result (v2.0)
+// const searchLetter = async(e) => {
+//   fetchMeals();
+//   return e.key;
+// };
+// const input = document
+//   .getElementById('input')
+//   .addEventListener('keyup', searchLetter);
+// // const baseUrl = `https://www.themealdb.com/api/json/v1/1/search.php?f=${input}`;
+
+// // Add the popup to the enntire card (v2.0)
+// const commentPopUp = (card) => {
+//   card.addEventListener('click', () => {
+//     const popup = open('', 'Popup', 'width=800,height=700');
+//     popup.className = 'popup'; //class stying not working.
+//     popup.document.body.appendChild(card);
+//   });
+// }
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.